app/Http/Controllers/Admin/OrderController.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Http\Requests\Admin_OrderFormRequest;
+use App\Models\Order;
+use App\Models\User;
+
+class OrderController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $orders = Order::with('user:id,name')->paginate(config('app.records_per_page'));
+
+        return view('admin.orders.index', compact('orders'));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        $userIds = User::select('id', 'name', 'phone_number')->get();        
```

Comment:
đặt lại tên biến nhé
```suggestion
        $users = User::select('id', 'name', 'phone_number')->get();        
```